app/Helpers/AvatarHelper.php

```diff
@@ -11,21 +11,39 @@ class AvatarHelper
     /**
      * Generate a new random avatar.
      *
-     * The Multiavatar library takes a name to generate a unique avatar.
+     * If the user has an avatar_style value set from his settings page the Dicebear
+     * avatar library will be used, otherwise it defaults to Multiavatar.
+     *
+     * The Multiavatar and Dicebear libraries take a name to generate a unique avatar.
      * However, contacts can be created in Monica without a name. When this case
      * happens, we'll generate a fake name for the contact, and generate an avatar
      * based on that name.
      */
-    public static function generateRandomAvatar(Contact $contact): string
+    public static function generateRandomAvatar(Contact $contact, $avatarStyle = null): array
     {
-        $multiavatar = new MultiAvatar;
-
         if (is_null($contact->first_name)) {
             $name = Faker::create()->name();
         } else {
             $name = $contact->first_name.' '.$contact->last_name;
         }
 
-        return $multiavatar($name, null, null);
+        if ($avatarStyle) {
+            return [
+                'type' => Contact::AVATAR_TYPE_URL,
+                'content' => self::getDicebearLink($avatarStyle, $name),
```

Comment:
What if we use the id (uuid) so it's always the same for one contact?
```suggestion
                'content' => self::getDicebearLink($avatarStyle, $contact->id),
```